### PR TITLE
Update kubernetes-api.md for zh

### LINKED ISSUE #18809: Delete extensions/v1beta1 in docs
**This is a Feature Request**

Since v1.16, extensions/v1beta1 has been deleted and it should not be used in docs

**What would you like to be added**
The following pages should be edited due to this deletion.
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/overview/kubernetes-api.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/workloads/controllers/deployment.md
- [x] https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/workloads/controllers/replicaset.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/access-authn-authz/admission-controllers.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/kubectl/conventions.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/reference/using-api/api-overview.md
- [ ] https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/access-application-cluster/ingress-minikube.md
- [x] https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/manage-kubernetes-objects/declarative-config.md
- [x] https://github.com/kubernetes/website/blob/master/content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md

**Why is this needed**
extensions/v1beta1 is deleted in 1.16

**Comments**

### PR DESCRIPTION
**Pull request by huchengze**
This PR is going to fix #18809 .
update [kubernetes-api.md](https://github.com/kubernetes/website/blob/master/content/zh/docs/concepts/overview/kubernetes-api.md) for zh.
/language zh

### CODE REVIEW

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -226,40 +226,41 @@ There are two supported paths to extending the API with [custom resources](/docs
...
 
-## 启用 API 组
+## 启用或禁用 API 组
 
 某些资源和API组默认情况下处于启用状态。 可以通过在apiserver上设置 `--runtime-config` 来启用或禁用它们。
 `--runtime-config` 接受逗号分隔的值。
 例如：要禁用batch/v1，请设置 `--runtime-config=batch/v1=false`，以启用batch/v2alpha1，请设置`--runtime-config=batch/v2alpha1`。
 该标志接受描述apiserver的运行时配置的逗号分隔的一组键值对。
 
-重要：启用或禁用组或资源需要重新启动apiserver和控制器管理器来使得 `--runtime-config` 更改生效。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/overview/kubernetes-api.md:247]: https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/overview/kubernetes-api.md#enabling-or-disabling-api-groups

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 2]
```diff
@@ -226,40 +226,41 @@ There are two supported paths to extending the API with [custom resources](/docs
...
+For example: to enable deployments and daemonsets, set
+`--runtime-config=extensions/v1beta1/deployments=true,extensions/v1beta1/daemonsets=true`.
+
+Individual resource enablement/disablement is only supported in the `extensions/v1beta1` API group for legacy reasons.
 -->
 
-## 启用组中资源
+## 启用 extensions/v1beta1 组中具体资源
+
+在 `extensions/v1beta1` API组中，DaemonSets，Deployments，StatefulSet, NetworkPolicies, PodSecurityPolicies 和 ReplicaSets是默认禁用的。
```
Comment (tanjunchen) [inline content/zh/docs/concepts/overview/kubernetes-api.md:261]:
```suggestion
在 `extensions/v1beta1` API 组中，DaemonSets、Deployments、StatefulSet、NetworkPolicies、PodSecurityPolicies 和 ReplicaSets 默认是禁用的。
```

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 3]
```diff
@@ -226,40 +226,41 @@ There are two supported paths to extending the API with [custom resources](/docs
...
+例如：要启用 deployments 和 daemonsets，请设置 `--runtime-config=extensions/v1beta1/deployments=true,extensions/v1beta1/daemonsets=true`。
 
-DaemonSets，Deployments，HorizontalPodAutoscalers，Ingress，Jobs 和 ReplicaSets是默认启用的。
-其他扩展资源可以通过在apiserver上设置 `--runtime-config` 来启用。
-`--runtime-config` 接受逗号分隔的值。 例如：要禁用 Deployment 和 Ingress，
-请设置 `--runtime-config=extensions/v1beta1/deployments=false,extensions/v1beta1/ingress=false`
+{{< note >}}出于遗留原因，仅在`extensions / v1beta1` API组中支持各个资源的启用/禁用。{{< /note >}}
```
Comment (tanjunchen) [inline content/zh/docs/concepts/overview/kubernetes-api.md:264]: API 组 
中英文之间是需要空格的

File: content/zh/docs/concepts/overview/kubernetes-api.md [hunk 4]
```diff
@@ -226,40 +226,41 @@ There are two supported paths to extending the API with [custom resources](/docs
...
-## 启用 API 组
+## 启用或禁用 API 组
 
 某些资源和API组默认情况下处于启用状态。 可以通过在apiserver上设置 `--runtime-config` 来启用或禁用它们。
 `--runtime-config` 接受逗号分隔的值。
 例如：要禁用batch/v1，请设置 `--runtime-config=batch/v1=false`，以启用batch/v2alpha1，请设置`--runtime-config=batch/v2alpha1`。
 该标志接受描述apiserver的运行时配置的逗号分隔的一组键值对。
 
-重要：启用或禁用组或资源需要重新启动apiserver和控制器管理器来使得 `--runtime-config` 更改生效。
+{{< note >}}启用或禁用组或资源需要重新启动apiserver和控制器管理器来使得 `--runtime-config` 更改生效。{{< /note >}}
```
Comment (tanjunchen) [inline content/zh/docs/concepts/overview/kubernetes-api.md:247]: 启动 apiserver
中英文之间是需要空格的

**Comment by tengqm** [comment]
@huchengze Got a few minutes to address the review comments?

**Comment by huchengze** [comment]
Sorry，I‘m a little busy these days.
I will close this pr and finish it in the end of this month.

**Pull request closed without merging**